apps/rush-lib/src/logic/pnpm/PnpmLinkManager.ts

```diff
@@ -222,11 +222,17 @@ export class PnpmLinkManager extends BaseLinkManager {
     //   C%3A%2Fwbt%2Fcommon%2Ftemp%2Fprojects%2Fapi-documenter.tgz
     //   C%3A%2Fdev%2Fimodeljs%2Fimodeljs%2Fcommon%2Ftemp%2Fprojects%2Fpresentation-integration-tests.tgz_jsdom@11.12.0
     //   C%3A%2Fdev%2Fimodeljs%2Fimodeljs%2Fcommon%2Ftemp%2Fprojects%2Fbuild-tools.tgz_2a665c89609864b4e75bc5365d7f8f56
-    const folderNameInLocalInstallationRoot: string =
+    let folderNameInLocalInstallationRoot: string =
       uriEncode(Text.replaceAll(absolutePathToTgzFile, path.sep, '/')) + folderNameSuffix;
```

Comment:
```suggestion
Path.convertToSlashes(absolutePathToTgzFile)
```
Mind, that function itself needs to be changed to `.replace(/\\/g, '/')` instead of using `String.prototype.split()`, but that's out of scope.